aws_prototype/main.py

```diff
@@ -125,34 +174,48 @@ def up():
     os.system(f"""copilot svc deploy --name viewer-svc --env {ENV_NAME}""")
     os.system(f"""copilot svc deploy --name amp-svc --env {ENV_NAME}""")
 
-    # if env_exist is False:
-    #     bucket: str = get_copilot_s3_bucket()
-    #     sync_command = f"aws s3 sync s3://{BAKCUP_DB}/ s3://{bucket}/"
-    #     os.system(sync_command)
-    #     command = "python aws_prototype/ecs_prepare_db.py"
-    #     copilot_exec_cmd = f"""copilot svc exec -a {APP_NAME} -e {ENV_NAME} -n amp-svc --command "{command}" """
-    #     os.system(copilot_exec_cmd)
+    if env_exist is False:
+        bucket: str = get_copilot_s3_bucket()
+        sync_command = f"aws s3 sync s3://{BACKUP_DB}/ s3://{bucket}/"
+        os.system(sync_command)
+        command = "python aws_prototype/ecs_prepare_db.py"
+        copilot_exec_cmd = f"""copilot svc exec -a {APP_NAME} -e {ENV_NAME} -n amp-svc --command "{command}" """
+        os.system(copilot_exec_cmd)
+        create_burner_account()
 
-    #     # Make dummy account
-    #     create_dummy_account(
-    #         app_name=APP_NAME,
-    #         env_name=ENV_NAME,
-    #     )
+    restore_copilot_prod_settings()
 
 
 def down():
     print(">>> Breaking down Copilot prototype")
+    switch_cp_apps()
+    restore_prototype_env_file()
     bucket: str = get_copilot_s3_bucket()
     s3 = boto3.resource("s3")
     bucket = s3.Bucket(bucket)
     bucket.objects.all().delete()
+    print(">>> Deleted all objects in S3")
     os.system("copilot app delete --yes")
+    restore_copilot_prod_settings()
 
 
 if __name__ == "__main__":
+    client = boto3.client("sts")
+    account_id = client.get_caller_identity()["Account"]
+    if account_id != "144664177605":
```

Comment:
Should this account id be hardcoded?

Reply:
TBH it's unlikely to change. And if it did there'd be bigger issues to worry about! 
Maybe stick it in a constant?